routes/open_pr_page.py

```diff
@@ -0,0 +1,14 @@
+import json
+
+import config
+import github_api
+
+
+def get_github_open_pr(reload_data):
+    if not config.GITHUB_PR_LIST.is_file() or reload_data:
+        github_api.get_open_pull_request()
```

Comment:
The call to `github_api.get_open_pull_request()` can also return the PRs so you don't always need to open the cache file.

Reply:
done